frontend/src/router/Router.tsx

```diff
@@ -20,6 +20,8 @@ export default function Router() {
       <Guard path="/" allowIf={!loggedIn} redirect="/home" exact>
         <Login {...{ setLoggedIn }} />
       </Guard>
+      <Guard path="/forgot" component={ForgotPassword} allowIf={true} exact />
```

Comment:
``<Guard allowIf={true}>`` defeats the purpose of using a Guard. You can use a normal Route to achieve the same behavior.
```suggestion
      <Route path="/forgot" component={ForgotPassword} exact />
```

Reply:
Ok thanks, wasn't sure at first